app/lib/applicant_case_details_section.rb

```diff
@@ -1,13 +1,11 @@
 class ApplicantCaseDetailsSection
   class << self
     def all_steps
-      %i[case_details partner applicant dependants dependant_details]
+      %i[applicant dependants dependant_details]
```

Comment:
I thought it was agreed that we would remove the dependants screen and ask the dependants question on the applicant screen?

Reply:
Ah the DoD on the ticket and the acceptance criteria are different on the issue of dependants question.